### DISCUSSION TITLE
[Enhancement] [form] Please add a method for form validation

### DISCUSSION THREAD

**Discussion by ZhaoKang0610** [element-plus/element-plus#15585] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `form`

## Description

Is it possible to provide a form validation method to validate the form input without displaying an error message below the input box, or is there another way to achieve this effect?

### Additional comments

_(empty)_

**Comment by chenxch** [comment]
So why not cancel the rule and use your own commit validation?
